frontstage/templates/passwords/password-expired.html

```diff
@@ -9,10 +9,8 @@ <h1 class="saturn">Your link has expired</h1>
 
 <div>
     <p>
-        You can either <a href="/passwords/forgot-password">request another password reset</a>.
-    </p>
-    <p>
-        or <a href="/sign-in">sign in</a> with your existing password.
+        You can <a href="{{ url_for('passwords_bp.resend_password_email_expired_token', token=token) }}">request another password reset email</a><br/>
```

Comment:
Remove line break, its not in the design

Reply:
You're right, it isn't but if the break isn't there then half of the telephone number wraps around to the next line which doesn't look right.  Agreed to move the break to move just the telephone number to the next line.